lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1463,19 +1454,68 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => true,
           :description => 'Distributed tracing lets you see the path that a request takes through your distributed system. Enabling distributed tracing changes the behavior of some New Relic features, so carefully consult the [transition guide](/docs/transition-guide-distributed-tracing) before you enable this feature.'
         },
+        :'distributed_tracing.sampler.root' => {
+          :default => 'default',
+          :public => true,
+          :type => String,
+          :allowed_from_server => false,
+          :allowlist => %w[default adaptive always_on always_off trace_id_ratio_based],
```

Comment:
What is behind the `default` value being `default` instead of just `adaptive`? Did we need to have `default` as a kept option per like the spec or something